src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1378,7 +1380,9 @@ impl Pump {
     fn calculate_displacement_from_required_flow(&self, required_flow: VolumeRate) -> Volume {
         if self.speed.get::<revolution_per_minute>() > 0. {
             let displacement = Volume::new::<cubic_inch>(
-                required_flow.get::<gallon_per_second>() * 231.0 * 60.0
+                required_flow.get::<gallon_per_second>()
+                    * Self::FLOW_CONSTANT_RPM_CUBIC_INCH_TO_GPM
+                    * 60.0
```

Comment:
**thought**: I guess `60.0` represents the seconds in a minute? If so, consider extracting to a const variable. Note that `60.0` is used later as well.